.github/workflows/raise-products.yml

```diff
@@ -41,10 +41,12 @@ jobs:
       - name: Raise Products
         run: |
           export engineUrl=${{ inputs.engineUrl }}
-          export IVY_JAVA_HOME=$JAVA_HOME_17_X64
+          export IVY_JAVA_HOME=$JAVA_HOME_21_X64
           echo "workDir=$(pwd)" >> $GITHUB_ENV
           ./raise-all-market-products.sh ${{ inputs.version }} ${{ inputs.product }}
         shell: bash
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
this workflow doesn't really run by now ... as we do not have a single technical octopus (market-org) user that is allowed to file PRs on all repos. I'd vote to introduce one somewhen ...
but for now the pipeline must be run manually:
you may check it out on a simply run `./raise-all-market-products.sh 12.0.0-SNAPSHOT deepl-connector`